package.json

```diff
@@ -41,6 +41,9 @@
     "author": "pxblue <pxblue@eaton.com>",
     "license": "BSD-3-Clause",
     "devDependencies": {
-        "codecov": "^3.8.1"
-    }
+        "@pxblue/prettier-config": "^1.0.3",
+        "codecov": "^3.8.1",
+        "prettier": "^2.3.0"
+    },
+    "prettier": "@pxblue/prettier-config"
```

Comment:
Does this do anything? There's no prettier script that runs at the root, it just jumps into the folders and runs it there I think:
![image](https://user-images.githubusercontent.com/29152776/118121388-689a4c00-b3bf-11eb-9a1b-fdb00dffe9ae.png)